src/main/kotlin/dev/rosewood/roseskyblock/feature/default/DefaultCommand.kt

```diff
@@ -12,7 +12,7 @@ class DefaultCommand(rosePlugin: RosePlugin) : SkyblockCommand(rosePlugin) {
 
     override fun create(manager: CommandManager<CommandSender>, builder: Command.Builder<CommandSender>) {
         builder.handler { context ->
-            val localeManager = this.rosePlugin.getManager(LocaleManager::class)
```

Comment:
this

Reply:
`this` is not going back there, you can put it back if you really want